### PR TITLE
cmd/importer: specify the config file

### PR DESCRIPTION
**Pull request by hanfei1991**
sepcify the config file and we can point out the stats file path. Prepare to generate data with stats.

### CODE REVIEW

**Comment by hanfei1991** [comment]
@shenli @coocood PTAL

File: cmd/importer/config.toml [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Importer Configuration.
 
+[ddl]
 table-sql = "create table t(a int primary key, b double, c varchar(10), d date unique, e time unique, f timestamp unique, g date unique, h datetime unique, i year);"
 
 index-sql = "create unique index u_b on t(b);"
 
+[stats]
+stats-file-path = "./t.json"
```
Comment (coocood) [inline cmd/importer/config.toml]: How about `./stats.json`?

Comment (hanfei1991) [reply cmd/importer/config.toml]: ok

**Comment by coocood** [comment]
LGTM

**Pull request merged**